cognite_toolkit/modules/cdf_data_pipeline_asset_default/datasets/dataset.yaml

```diff
@@ -0,0 +1,20 @@
+external_id: 'ds-asset:{{location_name}}'
+name:  'asset:{{location_name}}'
+description: 'Asset data for {{location_name}}'
+metadata: {
```

Comment:
Would {{source_name}} be relevant to add to the metadata as a label that might be useful?